### PR TITLE
default port for Octant is now 7777

### LINKED ISSUE #185: Proposal: create a well defined port for the octant API
Problem:

Octant starts its API on a random port on 127.0.0.1. This avoids potential conflicts. Giving Octant a well defined port number by default means that you could know what the port number is and restarts of the application wouldn't require the browser URL to change. I'm going to suggest two numbers for default ports: 

* 102016 - date Heptio was founded
* 7777 - Kubernetes people seem to like 7s

### PR DESCRIPTION
**Pull request by wwitzel3**
fixes #185 
Signed-off-by: Wayne Witzel III <wayne@riotousliving.com>

### CODE REVIEW

**Comment by wwitzel3** [comment]
fixes #185

**Comment by GuessWhoSamFoo** [comment]
What should the new behavior be when the port is occupied?

**Comment by wwitzel3** [comment]
We can exit with an error or continue with a warning. I think we should exit with an error if the requested listening port is not available. We can direct the uses to the OCTANT_LISTENER_ADDR setting.

File: internal/dash/dash.go [hunk 1]
```diff
@@ -282,7 +283,12 @@ func initModuleManager(options *moduleOptions) (*module.Manager, error) {
 
 func buildListener() (net.Listener, error) {
 	listenerAddr := api.ListenerAddr()
-	return net.Listen("tcp", listenerAddr)
+	conn, err := net.DialTimeout("tcp", listenerAddr, time.Second*5)
```
Comment (bryanl) [inline internal/dash/dash.go]: Couldn’t the timeout be < than 500ms? 5 seconds would slow dow octant’s start up

Comment (wwitzel3) [reply internal/dash/dash.go]: yeah, I'll make it 500ms that should be plenty of time

**Pull request merged**